.github/workflows/release.yml

```diff
@@ -2,28 +2,37 @@ name: Deploy Main
 on:
   release:
     types: [published]
+
+env:
+  PYTHON_VERSION: "3.12"
+
 jobs:
   deploy-pypi:
     runs-on: ubuntu-latest
-    environment: deployment
```

Comment:
we need this line for branch protection rules to find/pass the test evironment

Reply:
fixed